doc/man/8/ceph.rst

```diff
@@ -411,7 +411,13 @@ Subcommand ``tell`` sends command to particular mds.
 
 Usage::
 
-	ceph mds tell <who> <args> [<args>...]
+	ceph mds tell <role> <args> [<args>...]
```

Comment:
`mds tell` is now obsolete. This should be replaced with `ceph tell <who>` where `<who>` may be `mds.*` or `mds.<name>`. (It does not take a role.)

Reply:
dropped `mds tell` and fixed `ceph tell` with the correct arguments displayed in the implementation.